src/app.ts

```diff
@@ -58,6 +59,10 @@ async function app() {
       'specify which file types GoFSH should accept as input: json-only (default), xml-only, json-and-xml'
     )
     .option('--indent', 'output FSH with indented rules using context paths')
+    .option(
+      '--meta-profile <mode>',
+      'specify how meta.profile on Instances should be applied to the InstanceOf keyword: only-one (default), first, none'
+    )
```

Comment:
I think this needs to be documented in the README.md (as does `--indent`). And then both options need to be documented in FSHSchool as well.